profile.md

```diff
@@ -142,17 +140,16 @@ Note that when both the originating identity HTTP Header and the Context
 object appear in the same message the `platform` value MUST be the same
 for both.
 
-#### Cloud Foundry
+#### Cloud Foundry Context Object
 
 *`platform` Property Value*: `cloudfoundry`
 
 The following properties are defined for usage within a Cloud Foundry
 deployment:
 
-- `organization_guid`<br>
-  The organization GUID as defined by the Cloud Foundry
-  specification/project. This property MUST be a non-empty string serialized
-  as follows:
+- `organization_guid`  
+  The GUID of the organization that a Service Instance is associated with.
```

Comment:
The text here wasn't instance specific before, it was more generic. For now this is probably ok but we need to be careful that if "context" is ever used on messages that are not necessarily related to instances then we'll need to revert this change.